packages/core-mobile/app/services/send/SendService.ts

```diff
@@ -9,14 +9,14 @@ import { SendServicePVM } from 'services/send/SendServicePVM'
 import { RpcMethod } from 'store/rpc'
 import { getAddressByNetwork } from 'store/account/utils'
 import { TransactionParams as BtcTransactionParams } from 'store/rpc/handlers/bitcoin_sendTransaction/utils'
-import { TransactionParams as AvalancheTransactionParams } from 'store/rpc/handlers/avalanche_sendTransaction/utils'
 import { SendServiceAVM } from 'services/send/SendServiceAVM'
 import { transactionRequestToTransactionParams } from 'store/rpc/utils/transactionRequestToTransactionParams'
 import { type NftTokenWithBalance, TokenType } from '@avalabs/vm-module-types'
 import {
   getAvalancheCaip2ChainId,
   getEvmCaip2ChainId
 } from 'temp/caip2ChainIds'
+import { AvalancheTxParams } from 'store/rpc/handlers/avalanche_sendTransaction/avalanche_sendTransaction'
```

Comment:
we are deleting this handler so this type should actually live in the avalanche module

Reply:
moved to avalanche-module